polars/polars-core/src/chunked_array/object/mod.rs

```diff
@@ -2,10 +2,9 @@ pub mod builder;
 mod iterator;
 
 pub use crate::prelude::*;
-use crate::utils::arrow::array::ArrayData;
-use arrow::array::{Array, ArrayRef, BooleanBufferBuilder, JsonEqual};
+use arrow::array::*;
 use arrow::bitmap::Bitmap;
-use serde_json::Value;
```

Comment:
arrow2 does not depend on serde or json unless json IO is required. This reduces the dependency tree.